FirebaseAuth/Sources/Backend/RPC/FIREmailLinkSignInResponse.h

```diff
@@ -49,6 +55,10 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property(nonatomic, assign) BOOL isNewUser;
 
+@property(nonatomic, strong, readonly, nullable) NSString *MFAPendingCredential;
```

Comment:
nit: Can you add this documentation:

```
/** @property MFAPendingCredential
    @brief An opaque string that functions as proof that the user has successfully passed the first factor check.
*/
```

Reply:
Of course I can. Should I add same comment in `FIRVerifyPasswordResponse`? [here](https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseAuth/Sources/Backend/RPC/FIRVerifyPasswordResponse.h#L69)
`FIRVerifyPasswordResponse` also has `MFAPendingCredential `  property but no document.

Reply:
Yes, that would be great! Can you also add the comment to `FIRVerifyAssertionResponse` as well:

https://github.com/firebase/firebase-ios-sdk/blob/3cb4d9ceae4bda421e5a7ebd050c64192b523515/FirebaseAuth/Sources/Backend/RPC/FIRVerifyAssertionResponse.h#L208